templates.json

```diff
@@ -114,8 +114,8 @@
             "debugSSHPort": "2225",
             "multiContainer": true,
             "support": "Community",
-            "status": "deprecated",
-            "customMessage": "Deprecated on templates update for Torizon 7",
```

Comment:
hey, one thing at a time. Let's first do it for TOS 6 and then you open for TOS 7 after the merge on bookworm.

Reply:
Forgot to remove it, will remove it